resources/assets/uad_lists.json

```diff
@@ -17939,11 +17939,10 @@
   {
     "id": "com.google.android.packageinstaller",
     "list": "Google",
-    "description": "Occasionally runs in the background.\nRelated to installing Google-specific packages? Possibly something to do with Play Services and back-end updates/installations. Installing an app through Google Play worked fine with it disabled.\nNOTE: Some Huawei devices bootloop after uninstalling/disabling this package and require a full restore via eRecovery. Be careful!",
```

Comment:
Why did you remove this part if I may ask? It was suggested by the user who created the issue (the "NOTE" part).